tests/rptest/tests/datalake/cluster_restore_test.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, test_context: TestContext):
             # Frequent Iceberg uploads.
             "iceberg_enabled": "true",
             "iceberg_catalog_commit_interval_ms": 1000,
+            "iceberg_target_lag_ms": 1000,
```

Comment:
still needed?

Reply:
Not needed per se, but when I wrote the test I had in mind that we were translating frequently, so seems like a reasonable inclusion